core/src/test/scala/config/InstanceClasses.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.cassandra
+package config
+
+import java.net.InetSocketAddress
+import java.util
+import java.util.concurrent.Executor
+
+import com.datastax.driver.core._
+import com.datastax.driver.core.exceptions.DriverException
+import com.datastax.driver.core.policies._
+import io.netty.channel.socket.SocketChannel
+import io.netty.handler.ssl.SslHandler
+
+class MyJavaExecutor extends Executor {
+  override def execute(command: Runnable): Unit = {}
+}
+
+class MyAddressTranslator extends AddressTranslator {
+  override def init(cluster: Cluster): Unit                             = {}
+  override def translate(address: InetSocketAddress): InetSocketAddress = null
```

Comment:
`null` values are ugly :D . You could define in your test utils class something like this:

```scala
class Null[A] { var t: A = _ }
object Null {
  def apply[A]: A = new Null[A].t
}
```
Then, you could initiate these null instances in a more elegant way:

```scala
override def translate(address: InetSocketAddress): InetSocketAddress = Null[InetSocketAddress]
```